packages/expo-doctor/src/checks/SupportPackageVersionCheck.ts

```diff
@@ -0,0 +1,41 @@
+import { warnAboutDeepDependenciesAsync } from '../utils/explainDependencies';
+import { getRemoteVersionsForSdkAsync } from '../utils/getRemoteVersionsForSdkAsync';
+import { gteSdkVersion } from '../utils/gteSdkVersion';
+import { DoctorCheck, DoctorCheckParams, DoctorCheckResult } from './checks.types';
+
+export class SupportPackageVersionCheck implements DoctorCheck {
+  description = 'Verifying prebuild support package versions are compatible';
+
+  async runAsync({ exp, projectRoot }: DoctorCheckParams): Promise<DoctorCheckResult> {
+    const advice: string[] = [];
+    const issues: string[] = [];
+
+    if (gteSdkVersion(exp, '45.0.0')) {
```

Comment:
Invert this, it's much easier to read "if less than 45 then return ..." makes it very clear at a glance that this test is +45. Might even make sense to add this type of filter to the `DoctorCheck` class so there's a unified system for bailing on SDK versions.

Reply:
really like the idea of integrating the filter.

Reply:
used LT for global check; individual test checks are moot or now b/c it only works on 46+

Reply:
@EvanBacon scratch that, you inspired me to integrate the version checking into the check structure :-). All tests run on compatible projects currently, but the version checking is under test.